src/main/java/core/basesyntax/FindNumbers.java

```diff
@@ -11,6 +10,21 @@
      * числа 92, 18, 26 и 0.</p>
      */
     public int[] getAllNumbers(String text) {
-        return null;
+        String[] temp = text.replaceAll("[^ 0-9-]", " ").split(" ");
+        int count = 0;
+        for (int i = 0; i < temp.length; i++) {
+            if (!temp[i].equals("") && !temp[i].equals(" ")) {
+                count++;
+            }
+        }
+        int[] fin = new int[count];
+        int ind = 0;
+        for (String s : temp) {
```

Comment:
several for loops are bad for performance, try to solve this task using only one